### ISSUE TITLE
questions

### ISSUE THREAD

**Issue by vasan123** [ibrahim4529/flask-chatbot#2]
good one

**Comment by irfanandratama** [comment]
Hey @vasan123 I saw your question [here](https://www.chatbots.org/ai_zone/viewthread/3405/). We have same trouble, my bot cannot detect topic tag when in Flask. Do you know the solution yet?

**Comment by vasan123** [comment]
Nope, I have written my own parser for that purpose, but however its not same as topic.

**Comment by irfanandratama** [comment]
Well I think I found the solution. You could check at my fork or the pull request of this repo.
